processing/src/main/java/io/druid/segment/filter/BoundFilter.java

```diff
@@ -161,7 +165,15 @@ public boolean supportsBitmapIndex(BitmapIndexSelector selector)
     return Filters.bitmapsFromIndexes(getBitmapIndexIterator(boundDimFilter, bitmapIndex), bitmapIndex);
   }
 
-  private static Iterable<Integer> getBitmapIndexIterator(
+  private static IntList getBitmapIndexList(
+      final BoundDimFilter boundDimFilter,
+      final BitmapIndex bitmapIndex
+  )
+  {
+    return IntIteratorUtils.toIntList(getBitmapIndexIterator(boundDimFilter, bitmapIndex).iterator());
```

Comment:
Thanks. I added a new util class ```IntListUtils``` for this class, but am not sure about its location..